vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileFragment.kt

```diff
@@ -185,12 +185,9 @@ class RoomProfileFragment @Inject constructor(
         is RoomListQuickActionsSharedAction.NotificationsAllNoisy -> {
             roomProfileViewModel.handle(RoomProfileAction.ChangeRoomNotificationState(RoomNotificationState.ALL_MESSAGES_NOISY))
         }
-<<<<<<< HEAD
-=======
         is RoomListQuickActionsSharedAction.NotificationsAll -> {
```

Comment:
@Florian14 are you ok about this update ?

Reply:
@yostyle I don't know why this is still present in Element (I think that `NotificationsAll` and `NotificationsAllNoisy` have been merged into a single item. I cannot have both displayed in the UI for each type of room (I never have the noisy option but under the hood, it's probably the "noisy" which is set by selecting notifications all...).

At each rebase we delete this case but I am not sure this is still relevant.

Reply:
@yostyle  I confirm the description of @Florian14
only 3 states are used now in Element
<img width="965" alt="image" src="https://user-images.githubusercontent.com/8969772/181068166-9193eb6e-7994-4c6c-ad6d-39105752fbc2.png">
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/8969772/181069637-30f810a3-1ca4-4c90-8da4-50e48247e9ad.png">

- I think we should let this useless code be added during this rebase, and plan an action in Element-Android to remove this case